### PR TITLE
[BPK-1003] Add mobile scroll container docs

### PR DESCRIPTION
**Pull request by matthewdavidson**
localhost_8080_components_web_mobile-scroll-container

### CODE REVIEW

File: packages/bpk-docs/src/pages/ChipsPage/ChipsPage.js [hunk 1]
```diff
@@ -113,7 +113,6 @@ const ChipsPage = () => (
     ]}
     components={components}
     readme={chipReadme}
-    sassdocId="cards"
```
Comment (matteo-hertel) [inline packages/bpk-docs/src/pages/ChipsPage/ChipsPage.js]: is this change intentional? if so ignore this comment 😄

Comment (matthewdavidson) [reply packages/bpk-docs/src/pages/ChipsPage/ChipsPage.js]: Aye! This is the Chips page - that `sassdocId` is pointing to cards!! copypasta error.
